index.bs

```diff
@@ -552,18 +552,36 @@ The header's ABNF is:
 The 'Sec-CH-UA-Form-Factor' Header Field {#sec-ch-ua-form-factor}
 ------------------------------
 
-The <dfn http-header>`Sec-CH-UA-Form-Factor`</dfn> request header field gives a server information
-about the [=user agent=]'s [=form-factor=]. It is a [=Structured Header=] whose value MUST be a
-[=structured header/string=]. Its value SHOULD match one of the following common form-factor values:
-"Automotive", "Mobile", "Tablet", "TV", "VR", "XR", "Unknown" or the empty string.
-[[!RFC8941]].
+The <dfn http-header>`Sec-CH-UA-Form-Factor`</dfn> request header field gives a
+server information about the [=user agent=]'s [=form-factor=]. It is a
+[=Structured Header=] whose value MUST be a [=structured header/list=]
+[[!RFC8941]]. Its values SHOULD describe the form-factor of the device using
+one or more of the following common form-factor values: "Desktop",
+"Automotive", "Mobile", "Tablet", "TV", "VR", or "XR". Order of the values
+in the list is not significant.
+
+<div class="note" heading="">
+The form-factor of a device describes how the user interacts with the
+device.
+ * "Desktop" refers to a user-agent running on a personal computer.
+ * "Automotive" refers to a user-agent embedded in a vehicle, where the user
+    may be responsible for operating the vehicle and unable to attend to small
+    details.
+ * "Mobile" refers to small, touch-oriented device typically carried on a
+    user's person.
+ * "Tablet" refers to a touch-oriented device larger than "Mobile" and not
+    typically carried on a user's person.
+ * "TV" refers to a large, multi-user device desiged primarily for viewing videos.
+ * "VR" refers to an immersive, gesture-oriented device.
+ * "XR" is similar to "VR"  but includes devices that integrate with the
+    environment around the user.
```

Comment:
I'm not really sure how to distinguish these two. Maybe this is an example where the list would be useful ("XR" for XR things that aren't VR goggles and "XR, VR" for things that are VR goggles).

Reply:
I think that's one way we could add it in if we hear of use cases for "just VR". We could start conservative like @nielsbasjes suggests at first, WDYT?